test/fixtures/bundle.js

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
This file should not be commited.

Reply:
Thanks to these suggestions, the new code has been submitted